packages/typespec-ts/test/modularUnit/scenarios/operations/operations.md

```diff
@@ -768,12 +801,16 @@ export async function _testDeserialize(
   return barDeserializer(result.body);
 }
 
-export async function test(
+export function test(
   context: Client,
-  options: TestOptionalParams = { requestOptions: {} }
-): Promise<Bar> {
-  const result = await _testSend(context, options);
-  return _testDeserialize(result);
+  options: TestOptionalParams = { requestOptions: {} },
+): PagedAsyncIterableIterator<void> {
```

Comment:
what is the expected behavior given the typespec is 
```
@pagedResult
model Bar {
    lists: string[];
}
@post
op test(): Error | Bar;
```

Reply:
as discussed, we will do no more things for now, until we have further info from https://github.com/Azure/typespec-azure/issues/1993